docs/information/supported_adapters.md

```diff
@@ -29,6 +29,22 @@ Zigbee2MQTT officially supports the following adapters:
     <td><a href="https://slae.sh/projects/cc2652/#flashing">Flashing</a></td>
     <td><a href="https://slae.sh/projects/cc2652/">Slae.sh</a></td>
   </tr>
+  <tr>
+    <td><img src="../images/Tubes_USB_Coordinator_CC2652P2_variant.png"></td>
+    <td>Tube's CC2652P2 USB Coordinator</td>
+    <td>Open source hardware CC2652P based USB connected adapter with external antenna and USB extension cable</a></td>
+    <td><a href="https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin">Coordinator</a><br/> <a href="https://github.com/Koenkk/Z-Stack-firmware/raw/master/router/Z-Stack_3.x.0/bin/">Router</a></td>
```

Comment:
Please link directly to the file, not the folder.

Reply:
Done